### DISCUSSION TITLE
Some repos showing requested or daily when should be hourly

### DISCUSSION THREAD

**Discussion by halljus** [renovatebot/renovate#22901] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

N/A

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I have many repos all with the same renovate.json config and all but two of them are working as expected, but those two are not being renovated hourly. When I look in the Mend developer dashboard, the "Reason" column for the recent jobs includes a variety of either "requested" or "daily" for these repos (where the working repos all show "hourly"). And one of them doesn't quite seem to even be running daily, but that's possibly beside the point.

These are internal repos, so I can't link to them here, but I can provide the latest job ID from each if that helps:

0188d0fc-c789-76c5-9b67-8ac46b4679a0 (repo 1)
0188d0fc-d2ec-77fc-82f2-455cd397ffc0 (repo 2)

Any help would be most appreciated. Thanks.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by nabeelsaabna** [comment, marked as answer]
hi @halljus 
both repositories are with `status: onboarded` -> only daily jobs are getting scheduled. 
the remaining repositories have `status: activated` -> hourly jobs scheduled.

you can verify this by looking at the log message:
```
{
	"name": "renovate",
	"repository": "***-340B",
	"msg": "Repository result: done, status: onboarded, enabled: true, onboarded: true",
	"time": "2023-06-20T05:59:09.305Z"
}
```

**Reply by halljus** [reply]
Thank you for the response @nabeelsaabna. How would I go about changing the status to activated on these repos? I tried removing them from the Renovate GitHub application and adding them back in, but that didn't seem to do anything.

**Reply by rarkins** [reply]
You need to merge at least one update PR

**Reply by halljus** [reply]
Hmm, I initialized these repos with automerge enabled for minor and patch version updates, so there are no PRs and won't be until a major version of a dependency is released (updates are currently just sitting in their respective Renovate branches, not being automerged nor opening PRs). I suppose I could change the Renovate config to disable automerge, let it open a PR, and merge that?

Did I do something wrong to cause this situation where the repos are not activated? I did it the way I've done all my previous repos and haven't had a problem with any of those, but perhaps something has changed?

Thanks Rhys.

**Reply by rarkins** [reply]
You didn't do anything wrong. Nothing is wrong. Daily is acceptable for a free service in the first place, but you know what you need to do if you want it more frequently

**Reply by halljus** [reply]
Understood, thanks again.

**Reply by HonkingGoose** [reply]
For future reference, read the [Renovate docs, Known limitations](https://docs.renovatebot.com/known-limitations/) to learn about time/schedule based limitations of the hosted app.
